src/model/answer.js

```diff
@@ -0,0 +1,109 @@
+
+import { EventEmitter } from "events";
+
+class Answer extends EventEmitter {
+    constructor() {
+        super();
+        this.state = {
+            answers: [{
+                answerId: 0,
+                userId: "1",
+                questionId: 1,
+                text: "and perfect",
+            }, {
+                answerId: 1,
+                userId: "3",
+                questionId: 2,
+                text: "and boring",
+            }, {
+                answerId: 2,
+                userId: "1",
+                questionId: 0,
+                text: "and new.",
+            }],
+
+            currentId: 2,
+
+            //object used to fill the values from the input
+            newAnswer: {
+                answerId: "",
+                userId: "",
+                questionId: "",
+                text: "",
+            },
+
+        };
+    }
+
+
+    addAnswer(userId, questionId, text) {
+        this.state = {
+            ...this.state,
+            currentId: ++this.state.currentId,
```

Comment:
++ modifies the value in place btw :) The whole purpose of creating a new object was to not change the old once (so you should simply do +1 and not ++).